src/components/SidebarMobileView.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import DescriptionOutlinedIcon from "@material-ui/icons/DescriptionOutlined";
+import ArchiveIcon from "@material-ui/icons/Archive";
+import UnarchiveIcon from "@material-ui/icons/Unarchive";
+import DeleteOutlineIcon from "@material-ui/icons/DeleteOutline";
+
+import SidebarRow from "./SidebarRow";
+// const SidebarMobileView = () => {
+//   <div className=" sticky top-0 z-50 bg-black shadow-sm">
+//     <div className="flex w-full items-center justify-between mx-auto py-3 px-6 ">
+//       <SidebarRow
+//         Icon={DescriptionOutlinedIcon}
+//         title={"Notes"}
+//         path={"notes"}
+//       />
+//       <SidebarRow Icon={ArchiveIcon} title={"Archive"} path={"archive"} />
+//       <SidebarRow Icon={DeleteOutlineIcon} title={"Trash"} path={"trash"} />
+//     </div>
+//   </div>;
+// };
+
+// export default SidebarMobileView;
```

Comment:
You can remove the commented code if it doesn't have any future use 👍🏻 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

Reply:
will implement all the suggested code changes.